src/cmd/manage_nsfs.js

```diff
@@ -103,9 +103,7 @@ async function main(argv = minimist(process.argv.slice(2))) {
                 ...(ManageCLIError.FS_ERRORS_TO_MANAGE[err.code] ||
                 ManageCLIError.RPC_ERROR_TO_MANAGE[err.rpc_code] ||
                 ManageCLIError.InternalError), cause: err });
-        process.stdout.write(manage_err.to_string() + '\n', () => {
-            process.exit(1);
-        });
+        write_stdout_response(manage_err);
```

Comment:
let's avoid mixing errors with valid responses
in the CLI errors are written using throw_cli_error() while this function also writes events 
write_stdout_response() are for valid responses

Reply:
I think we can create one more function to print/record/handle events for errors like `write_stdout_errors()` and in `throw_cli_error()` we can just throw the error instead of handling event. And we can call this function in catch block which also handles the `InternalError` if occurs. wdyt?